qa/rpc-tests/test_framework/electrumutil.py

```diff
@@ -14,6 +20,56 @@ def compare(node, key, expected, is_debug_data = False):
 
 def bitcoind_electrum_args():
     import random
+    global ELECTRUM_PORT
+    ELECTRUM_PORT = random.randint(40000, 60000)
     return ["-electrum=1", "-debug=electrum", "-debug=rpc",
-            "-electrum.port=" + str(random.randint(40000, 60000)),
+            "-electrum.port=" + str(ELECTRUM_PORT),
             "-electrum.monitoring.port=" + str(random.randint(40000, 60000))]
```

Comment:
not related to this PR, cause it doesn't change the ports picking logic, buy couldn't we get port collision once in 20K times?

Reply:
Yes, this kind of sucks. Ideally, this should share port seeder framework with whats in `util.py`. But with the remapping feature it has, this looks like quite a bit of work.